src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const defaultCost = days * 40;
+
+  if (days >= 7) {
+    return defaultCost - 50;
```

Comment:
It is better to create variables for discount

Reply:
agree
all these numbers should be in variables, keep your code without "magic" numbers, I don't know what numbers 40, 20, 50 mean, you must explain it to me in the code